unionify/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+//! [UnionSize] is a derivable trait that calculates the size of union part
```

Comment:
Maybe add to this comment that it's the size specifically of a `repr(C)` union,?